platform-api/tests/test_discord.py

```
@@ -0,0 +1,111 @@
+        self.store = {"discord-webhook:announcements": "http://fake-discord.url"}
+        
+    async def get(self, key):
+        return self.store.get(key)
+        
+    async def put(self, key, value, expirationTtl=None):
+        self.store[key] = value
+
+@pytest.fixture
+def client(monkeypatch):
+    # Mock the discord client so it never makes real HTTP requests
+    async def fake_post_webhook(url, embed):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `fake_post_webhook`

Add return type annotation: `bool`

(ANN202)